### DISCUSSION TITLE
Renovate bot is updating wrapper/gradle-wrapper.properties

### DISCUSSION THREAD

**Discussion by Neenu1995** [renovatebot/renovate#12524] [Request Help]
Renovate is suggesting updates for [wrapper/gradle-wrapper.properties](https://github.com/googleapis/gapic-generator-java/pull/713/files), which is not ideal behaviour.  Changing a build tool version is not as simple as changing a dependency version. This PR accidentally got merged and broke client library builds.

Can we avoid suggesting updates to gradle-wrapper.properties files?

@vam-google

**Comment by viceice** [comment]
You should not blindly allow automerge without good ci checks. 😉


Check the docs for how about disable managers or automerge.

Btw I'm using renovate to update Gradle wrapper for my Android projects without any issues.
